pandas/tests/io/test_parquet.py

```diff
@@ -474,6 +474,19 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa):
         check_round_trip(df_compat, pa,
                          path='s3://pandas-test/pyarrow.parquet')
 
+    def test_partition_cols_supported(self, pa, df_full):
+        partition_cols =['bool','int']
+        df = df_full
+        path="a.txt"
```

Comment:
We use temporary files so that tests don't litter the current directory. Something like

```
    with tm.ensure_clean() as path:
        df.to_parquet(path, engine=pa, ...)
```